hooks/useVercelChat.ts

```diff
@@ -117,12 +124,20 @@ export function useVercelChat({ id, initialMessages }: UseVercelChatProps) {
     window.history.replaceState({}, "", `/chat/${id}`);
   };
 
-  const handleSendMessage = async () => {
+  const handleSendMessage = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
```

Comment:
why do you need to `preventDefault`?

Reply:
`handleSendMessage` is attached to the form's onSubmit handler. It triggers a GET request on submit. (old ways of submitting data through an HTML form, method='GET')

As we are managing everything on the client side (React app) so we are using event.preventDefault() function to prevent form's default behavior (from making a GET request, causing page reload with URL params)

so that's the reason. I can test it by removing, see how it behave